attr/xattr/attribute.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+// TODO: Just putting here for the moment to get rid of import cycle.
```

Comment:
As per the TODO. Interested in hearing opinions on where this interface would best be located.